rl_loop.py

```diff
@@ -104,6 +123,8 @@ def gather():
 
 
 def train(logdir=None):
+    while not enough_games_to_train():
+        time.sleep(60 * 5)
```

Comment:
How about we make the sleep-time configurable or a constant

Reply:
actually, the right thing to do is exit -- otherwise the gather step won't actually be repeated.